sdk/resourcemanager/monitor/armmonitor/build.go

```diff
@@ -2,6 +2,6 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 // This file enables 'go generate' to regenerate this specific SDK
-//go:generate pwsh ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate resourcemanager/monitor/armmonitor
+//go:generate pwsh ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate -removeUnreferencedTypes resourcemanager/monitor/armmonitor
```

Comment:
Is `-removeUnreferencedTypes` removing more than just what's associated with the Metrics Client? If it's removing other unused models, that's an additional breaking change and we should note it in the changelog.

Reply:
Yes. It will also remove unreferenced types such as base model only inherited by other models, error response models, etc. Because unreferenced types should not be used by any customers, we decide not to put such change in the breaking change changelog to prevent misleading. And Jeff also agrees not to treat such changes as breaking changes and we have released other stable modules with minor version bump with such removal.